.gitignore

```diff
@@ -1,7 +1,6 @@
 /rollup/build/
 /docs/components/api.json
-/dist/mapbox-gl-dev.js
```

Comment:
Isn't this change tied to #6513?

Reply:
I duplicated it here because the style-spec build changes also introduce new stuff into `dist/`